collect/cnrfc/cnrfc.py

```diff
@@ -496,6 +547,122 @@ def get_ensemble_forecast_watershed(watershed, duration, date_string, acre_feet=
+
+    # extract CSV from zip object
+    if get_web_status(url):
+        try:
+            csvdata = _get_forecast_csv(url)
+        except zipfile.BadZipFile:
+            print(f'ERROR: forecast for {date_string} has not yet been issued.')
+            raise zipfile.BadZipFile
+    
+    # raise error if user supplied an actual date string but that forecast doesn't exist
+    elif _date_string is not None:
+        print(f'ERROR: forecast for {date_string} has not yet been issued.')
+        raise zipfile.BadZipFile
+
+    # get date/time stamp from ensemble download page
+    time_issued = get_watershed_forecast_issue_time(duration, watershed, date_string)
+
+    # set path for case where path set to None
+    if path is None:
+        path = '{}/{}'.format(os.getcwd(), 
+                                url.split('/')[-1].replace('.zip','.csv'))
+
+    # write csvdata to specified path
+    path = path.replace('/', os.sep)
+    with open(path, 'wb') as f:
+        f.write(csvdata.read())
+
+def get_watershed_forecast(watershed, date_string, forecast_type, duration='daily', path=None):
+    """
+    from: get_watershed_ensemble_issue_time
+          get_watershed_ensemble_daily
+
+    download seasonal outlook for the watershed as zipped file, unzip, save as csv to path
+
+    Arguments:
+        watershed (str): the forecast group identifier
+        date_string (str): the forecast issuance date as a YYYYMMDDHH formatted string
+        forecast_type (str): one of deterministic or ensemble
+        duration (str): forecast data timestep (hourly or daily)
+        path (str): file name including file path/s3 key
+    Returns:
+        (dict): dictionary with data (dataframe) entry and info metadata dict
+    """
+    units = 'kcfs'
+
+    duration = _validate_duration(duration)
+
+    # store original date_string
+    _date_string = date_string
+
+    # forecast datestamp prefix
+    date_string = _default_date_string(date_string)
+
+    # define URL
+    url_end = ''
+
+    if forecast_type == 'deterministic':
+        url_end = '{0}_{1}_csv_export.zip'.format(date_string, watershed)
+    elif forecast_type == 'ensemble':
+        url_end = '{0}_{1}_hefs_csv_{2}.zip'.format(date_string, watershed, duration)
+    
+    url = 'https://www.cnrfc.noaa.gov/csv/' + url_end
+
+    # extract CSV from zip object
+    if get_web_status(url):
+        try:
+            csvdata = _get_forecast_csv(url)
+        except zipfile.BadZipFile:
+            print(f'ERROR: forecast for {date_string} has not yet been issued.')
+            raise zipfile.BadZipFile
+    
+    # raise error if user supplied an actual date string but that forecast doesn't exist
+    elif _date_string is not None:
+        print(f'ERROR: forecast for {date_string} has not yet been issued.')
+        raise zipfile.BadZipFile
+
+    # set path for case where path set to None
+    if path is None:
+        path = '{}/{}'.format(os.getcwd(), 
```

Comment:
What happens if someone provides a path like `path = 'C:/Users/narlesky/Desktop/2017021212_american_hefs_csv_hourly.csv'`?  Is the `os.getcwd()` necessary?

Reply:
If you put the full path, this condition does not apply and the downloaded file, if available, is dropped in 'C:/Users/narlesky/Desktop/2017021212_american_hefs_csv_hourly.csv'. 

os.getcwd() is what I used to drop the downloaded file in the same folder as the script using collect in the case where a path wasn't specified.

Reply:
I see.  How about this case?

script is stored at location: C:/Users/name/Desktop/folder/subfolder/script.py
```bash
$ cd C:/Users/name/Desktop
$ python folder/subfolder/script.py
```

Reply:
This one is strange. I can't find the file anywhere. It's not in the subfolder and it's not in the Desktop, even though that is the specified path.

Reply:
This is the behavior I would expect: can you confirm??
If the user does not provide a path name, the resulting files will be directly written to the desktop.

Reply:
Okay, that makes more sense. It performs as you would expect.

Reply:
Are you thinking we should do without this entirely and just always specify the path?